### PR TITLE
Add LocalDevice binding

### PR DESCRIPTION
**Pull request by TimeEngineer**
depends on #2439

### CODE REVIEW

File: oxidation/libparsec_python/libparsec/types.py [hunk 1]
```diff
@@ -152,6 +152,8 @@
         RevokedUserCertificate,
         TrustchainContext,
         freeze_time,
+        # LocalDevice
```
Comment (FirelightFlagboy) [inline oxidation/libparsec_python/libparsec/types.py:155]: _The enemies has been spotted_

File: oxidation/libparsec_python/libparsec/types.py [hunk 2]
```diff
@@ -307,4 +309,6 @@
     "RevokedUserCertificate",
     "TrustchainContext",
     "freeze_time",
+    # LocalDevice
```
Comment (FirelightFlagboy) [inline oxidation/libparsec_python/libparsec/types.py:312]: _The enemies has been spotted_

File: oxidation/libparsec_python/src/lib.rs [hunk 3]
```diff
@@ -180,5 +181,7 @@ fn _libparsec(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<certif::DeviceCertificate>()?;
     m.add_class::<trustchain::TrustchainContext>()?;
     m.add_function(wrap_pyfunction!(time::freeze_time, m)?)?;
+    // LocalDevice
```
Comment (FirelightFlagboy) [inline oxidation/libparsec_python/src/lib.rs:184]: _The enemies has been spotted_

**Review by FirelightFlagboy — CHANGES_REQUESTED** [review]

File: oxidation/libparsec_python/libparsec/types.py (hunk 1 above)
Comment (TimeEngineer) [reply oxidation/libparsec_python/libparsec/types.py:155]: Chief want this sorry :man_shrugging:

**Review by FirelightFlagboy — APPROVED** [review]

**Pull request merged**